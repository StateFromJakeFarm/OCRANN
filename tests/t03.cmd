  train_input2.txt train_output2.txt test_input2.txt test_output2.txt structure2.txt weights2.txt encoding.txt 0.01 100
