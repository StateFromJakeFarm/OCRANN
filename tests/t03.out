-0.099977793666 0.080290107271 -0.059754491229 -0.079985018679 0.060173018154 -0.039808479951 0.060839535174 0.070123972195 -0.080497415452 0.079817922446 0.070639434337 0.080860005986 -0.029662079442 -0.039704183800 -0.060750756960 0.060367494986 -0.049647682933 -0.049319098745 0.070193045124 -0.080687255641 
2
2
2
2
2
2
2
2
2
2
2
2
2
2
2
2
2
2
2
2
2
2
2
2
2
2
2
2
2
2
2
2
2
2
2
2
2
2
2
2
2
2
2
2
2
2
2
2
2
2
2
2
2
2
2
2
2
2
2
2
2
2
2
2
2
2
2
2
2
2
2
2
2
2
2
2
2
2
2
2
2
2
2
2
2
2
2
2
2
2
2
2
2
2
2
2
2
2
2
2
0.350000000000
